modules/ml/src/main/java/org/apache/ignite/ml/math/util/MatrixUtil.java

```diff
@@ -209,6 +213,92 @@ public static void unflatten(double[] fArr, Matrix mtx) {
                 mtx.setX(i, j, fArr[!isRowMode ? i * colsCnt + j : j * rowsCnt + i]);
     }
 
+    public static Matrix zipWith(Matrix mtx1, Matrix mtx2, IgniteBiFunction<Double, Double, Double> f) {
```

Comment:
Add explanation (zip is not a part of Java's world)

Reply:
Done.